src/FSharpLint.Core/Rules/Conventions/RaiseWithTooManyArguments/FailwithBadUsage.fs

```diff
@@ -116,7 +116,7 @@ let private runner (args: AstNodeRuleParams) =
                     | SynPat.Named (_, id, _, _, _) -> checkExpr app (Some id.idText)
                     | _ -> checkExpr app None)
                 | _ -> Array.empty
-        | _ -> Array.empty
+        // | _ -> Array.empty
```

Comment:
The newer sdks have this warning here
```
FSharpLint\src\FSharpLint.Core\Rules\Conventions\RaiseWithTooManyArguments\FailwithBadUsage.fs(119,11): error FS0026: This rule will never be matched
```
(which cause a build failure due to WarningsAsErrors being enabled)